### DISCUSSION TITLE
关于配置加密

### DISCUSSION THREAD

**Discussion by boycema** [hhyo/Archery#1465] [Ideas]
场景：一些大型企业对于服务器安全扫描，对文件内部的一些配置，密码不得以明文方式配置
现状：setting.py中的账号密码只能通过明文配置(不知道有其他实现加密配置的方式）
解决思路：可以参考java应用加密方式jasypt

望采纳

**Comment by LeoQuote** [comment]
你可以保存在别的地方,  settings.py 是 python 程序, 你自己在里面自己实现就好了, 从别的地方读, decrypt , 都可以做啊.
